src/BenchmarkDotNet/Parameters/ParameterInstance.cs

```diff
@@ -37,7 +37,7 @@ public string ToSourceCode()
                 ? parameter.ToSourceCode()
                 : SourceCodeHelper.ToSourceCode(value);
 
-        public string ToDisplayText(CultureInfo cultureInfo)
+        public string ToDisplayText(CultureInfo cultureInfo, int maxParameterColumnWidth)
```

Comment:
Yes, good idea!

Reply:
@JohannesDeml it's still `public`, have you forgot to change it?

Reply:
Oh, sorry! I changed `ToDisplayText(CultureInfo cultureInfo)` before (since that method is also not used outside of the class). Fixed access for `ToDisplayText(CultureInfo cultureInfo, int maxParameterColumnWidth)` now.